keps/sig-windows/20190418-windows-security-context.md

```diff
@@ -130,22 +202,38 @@ None
 
 ### Test Plan
 
-Unit tests will be added around `DetermineEffectiveSecurityContext` to ensure for each field in `WindowsSecurityOptions` the values in a container's `SecurityContext.WindowsOptions` can override values of the fields in the pod's `PodSecurityContext.WindowsOptions`.
+Unit tests will be added around `DetermineEffectiveSecurityContext` to ensure for each field in `WindowsOptions` the values in a container's `SecurityContext.WindowsOptions` can override values of the fields in the pod's `PodSecurityContext.WindowsOptions`.
 
-Overall test plans for GMSA are covered in the GMSA KEP.
+An e2e test case with `[sig-windows]` label will be added to the Windows test runs to exercise and verify RunAsUserName is correctly taking effect. The test will try to bring up a pod with a container based on NanoServer, default user `ContainerUser` and the entrypoint trying to execute a privileged operation like changing the routing table inside the container. The pod spec will specify `ContainerAdministrator` as the RunAsUserName to verify the privileged operation within the container can successfully execute. Without RunAsUserName working correctly, the default user in the container will fail to execute the privileged operation and the container will exit prematurely.
+
```

Comment:
^^ :) Glad we don't need a custom image. I forgot `nanoserver` already had both users defined.